.github/workflows/experimental-docker-build-test.yaml

```diff
@@ -120,6 +123,8 @@ jobs:
     secrets: inherit
     with:
       GIT_SHA: ${{ needs.determine-docker-build-metadata.outputs.gitSha }}
+      IMAGE_TAG: experimental_${{ needs.determine-docker-build-metadata.outputs.gitSha }}
+      FORGE_IMAGE_TAG: experimental_${{ needs.determine-docker-build-metadata.outputs.gitSha }}
```

Comment:
Why do you have to override FORGE_IMAGE_TAG btw if it's the same as IMAGE_TAG?

Reply:
I don't think it uses IMAGE_TAG. It goes and searches for the most recent commit hash which is not what I want. Please see below run.

https://github.com/aptos-labs/aptos-core/actions/runs/4288026830/jobs/7469878586